### PR TITLE
Add Refund files

### PR DESCRIPTION
**Pull request by julionc**
/cc @andrewpaliga @ch33sybr3ad

### CODE REVIEW

**Comment by andrewpaliga** [comment]
I actually don't know enough about this to review. @dominiquesr?

File: test/refund_test.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      :refund_line_items => [{ :line_item => 518995019, :quantity => 1 }]
+    )
+    assert_equal 703073504, refund.refund_line_items.first.line_item_id
+  end
+
+  test '#find should return a refund' do
+    fake "orders/450789469/refunds/509562969", :method => :get, :body => load_fixture('refund')
+    refund = ShopifyAPI::Refund.find(509562969, :params => {:order_id => 450789469})
+    assert_equal 509562969, refund.id
+  end
```
Comment (dominiquesr) [inline test/refund_test.rb]: Should there be a test for calculate as well?

File: test/fixtures/refund.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          "gift_card": false,
+          "name": "IPod Nano - 8gb - green",
+          "variant_inventory_management": "shopify",
+          "properties": [
+            {
+              "name": "Custom Engraving Front",
+              "value": "Happy Birthday"
+            },
+            {
+              "name": "Custom Engraving Back",
```
Comment (dominiquesr) [inline test/fixtures/refund.json]: I don't think you'd ever put Merry Christmas on the same item as Happy Birthday, but maybe I'm the only one that cares that this example doesn't make sense. ;)

**Comment by julionc** [comment]
You're completely right, @dominiquesr :sweat_smile:
Maybe, someone should update the json response from documentation. (See [refund#show](https://docs.shopify.com/api/reference/refund#show) & [order#show](https://docs.shopify.com/api/reference/order#show)).
BTW, i just removed Merry Christmas property, and added calculate method.

**Comment by dominiquesr** [comment]
LGTM but @kevinhughes27 has more experience in this repo.

**Comment by kevinhughes27** [comment]
I'm fine with this.

**Comment by julionc** [comment]
Bump

**Pull request merged**